lang/en.json

```diff
@@ -173,6 +173,7 @@
         "EncryptedSmtpTransport": "Enter the transport layer encryption required by your SMTP server.",
         "Error": "Error",
         "ErrorRequest": "Oops… there was a problem during the request. Maybe the server had a temporary issue, or maybe you requested a report with too much data. Please try again. If this error occurs repeatedly please %1$scontact your Matomo administrator%2$s for assistance.",
+        "ErrorRateLimit": "Oops… too many requests. This happens when you exceed the number of allowed requests to the API for a certain time period. See rate-limiting section in API documentation.",
```

Comment:
Could this perhaps contain a link the relevant documentation?

It reads a little better as "See the rate-limiting section in the API documentation."

Reply:
@bx80 i copy that sentence from the cloud UI ha, but there is no link or documentation for that. :(

Reply:
@peterhashair @sgiehl I'm thinking maybe we just remove this part for now? There's no point in directing people to documentation that doesn't exist and cloud should be able to override this message with a cloud specific link if needed.